zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -0,0 +1,100 @@
+//! Constants for Block Subsidy, Funding Streams, and Founders’ Reward
+
+use zebra_chain::parameters::Network;
+
+/// Whatever
```

Comment:
Please don't add placeholder comments, they disable the missing_docs clippy lint.

Instead, allow missing docs - it's easier to read and search for.

```suggestion
#[allow(missing_docs)]
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/1051/commits/9c2f8644f4e34af7d7ad2c1f4d2498fdcdc3f72e